game_quiz.py

```diff
@@ -0,0 +1,56 @@
+"""
+Welcome to Simple python quiz game
+"""
+import sys
+
+class Question:
+    """ The class based implementation of python quiz game"""
+    def __init__(self, prompt, answer):
+        self.prompt = prompt
+        self.answer = answer
```

Comment:
Single line break should be given here
```suggestion
        self.answer = answer
       
```